docs/make.jl

```diff
@@ -31,25 +29,285 @@ mathengine = MathJax3(Dict(
+            ),
+            MultiDocumenter.Link(
+                "aplavin/VLBIData",
+                "https://github.com/JuliaAPlavin/VLBIData.jl",
+            ),
+            MultiDocumenter.Link(
+                "aplavin/Difmap",
+                "https://github.com/JuliaAPlavin/Difmap.jl",
+            ),
+        ]
+    ),
+    MultiDocumenter.DropdownNav(
+        "Cosmology",
+        [
+            MultiDocumenter.MultiDocRef(
+                upstream = joinpath(clonedir, "Cosmology"),
+                path = "Cosmology",
+                name = "Cosmology",
+                giturl = "https://github.com/JuliaAstro/Cosmology.jl.git",
+            ),
+        ]
+    ),
+    MultiDocumenter.DropdownNav(
+        "Orbits & Ephemerides",
+        [
+            MultiDocumenter.MultiDocRef(
+                upstream = joinpath(clonedir, "Transits"),
+                path = "Transits",
+                name = "Transits",
+                giturl = "https://github.com/JuliaAstro/Transits.jl.git",
+            ),
+            MultiDocumenter.MultiDocRef(
+                upstream = joinpath(clonedir, "sefffal/PlanetOrbits"),
+                path = "sefffal/PlanetOrbits",
+                name = "sefffal/PlanetOrbits",
+                giturl = "https://github.com/sefffal/PlanetOrbits.jl.git",
+            ),
+            MultiDocumenter.MultiDocRef(
+                upstream = joinpath(clonedir, "EarthOrientation"),
+                path = "EarthOrientation",
+                name = "EarthOrientation",
+                giturl = "https://github.com/JuliaAstro/EarthOrientation.jl.git",
+            ),
+            MultiDocumenter.Link(
+                "JPLEphemeris",
+                "https://github.com/JuliaAstro/JPLEphemeris.jl",
+            ),
+            MultiDocumenter.MultiDocRef(
+                upstream = joinpath(clonedir, "AstroLib"),
+                path = "AstroLib",
+                name = "AstroLib",
+                giturl = "https://github.com/JuliaAstro/AstroLib.jl.git",
+            ),
+        ]
+    ),
+    MultiDocumenter.DropdownNav(
+        "Numerical Utilities",
+        [
+            MultiDocumenter.MultiDocRef(
+                upstream = joinpath(clonedir, "BoxLeastSquares"),
+                path = "BoxLeastSquares",
+                name = "BoxLeastSquares",
+                giturl = "https://github.com/JuliaAstro/BoxLeastSquares.jl.git",
+            ),
+            MultiDocumenter.MultiDocRef(
+                upstream = joinpath(clonedir, "LombScargle"),
+                path = "LombScargle",
+                name = "LombScargle",
+                giturl = "https://github.com/JuliaAstro/LombScargle.jl.git",
+            ),
+        ]
+    ),
+    MultiDocumenter.DropdownNav(
+        "Statistics",
+        [
+            MultiDocumenter.Link(
+                "sefffal/PairPlots",
+                "https://sefffal.github.io/PairPlots.jl/dev",
+            ),
+        ]
+    ),
+]
```

Comment:
This bit is all manual and gross for now just to stand things up, and can be replaced by something like #44 if we end up wanting to go the "single source of truth" route discussed there